src/views/TradingCompetition/components/ClaimModal/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react'
+import { useWeb3React } from '@web3-react/core'
+import styled from 'styled-components'
+import { Modal, Button, Flex, AutoRenewIcon, Heading, Text, Image } from '@pancakeswap-libs/uikit'
+import useI18n from 'hooks/useI18n'
+import { useTradingCompetitionContract } from 'hooks/useContract'
+import { useToast } from 'state/hooks'
+import { CompetitionProps } from '../../types'
+import NftBunnies from '../../pngs/cakers-nft.png'
+
+const ImageWrapper = styled(Flex)`
+  justify-content: center;
+  width: 100%;
+  height: fit-content;
+  img {
+    border-radius: ${({ theme }) => theme.radii.default};
+  }
+`
+
+const ClaimModal: React.FC<CompetitionProps> = ({ onDismiss, onClaimSuccess, userTradingInformation }) => {
```

Comment:
There is some placeholder stuff in this component. It's currently only needed for smart contract claim testing